app/jobs/course/user_deletion_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+class Course::UserDeletionJob < ApplicationJob
+  include TrackableJob
```

Comment:
Why does this need to be a `TrackableJob`?

Reply:
removed